File: frontend/lib/src/models/Post.dart
class Post {
  final String originalId;
  final String text;
  final String originalText;
  final String authorId;
  final String language;
  final DateTime postedAt;

  Post(this.originalId, this.text, this.originalText, this.authorId, this.language, this.postedAt);

  Post.fromMap(Map data) :
      originalId = data["original_id"],
      text = data["text"],
      originalText = data["original_text"],
      authorId = data["author_id"],
      language = data["language"],
      postedAt = data["posted_at"] != null? DateTime.parse(data["posted_at"]) : null;
}

File: backend/Domain/Models/Storage/Response/Post.cs
﻿using System;
using Newtonsoft.Json;

namespace Socneto.Domain.Models.Storage.Response
{
    public class Post
    {
        [JsonProperty("id")]
        public Guid PostId { get; set; }
        
        [JsonProperty("jobId")]
        public Guid JobId { get; set; }
        
        [JsonProperty("originalId")]
        public string OriginalId { get; set; }
        
        [JsonProperty("text")]
        public string Text { get; set; }
        
        [JsonProperty("originalText")]
        public string OriginalText { get; set; }
        
        [JsonProperty("authorId")]
        public string AuthorId { get; set; }
        
        [JsonProperty("language")]
        public string Language { get; set; }
        
        [JsonProperty("datetime")]
        public DateTime? PostedAt { get; set; }
    }
}